cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -528,7 +528,9 @@ def _deserialize_gate_op(
                 arg_function_language=arg_function_language,
                 required_arg_name=None,
             )
-            if isinstance(theta, (float, sympy.Basic)) and isinstance(phi, (float, sympy.Basic)):
+            if isinstance(theta, (int, float, sympy.Basic)) and isinstance(
```

Comment:
It might be nice to include other types such as the numpy integer and floating point types. It seems like we've run into this repeatedly and I don't think we have a good generic way to handle numeric types, so no problem if you'd like to defer to a future PR.

Reply:
Since this is in deserialization, we luckily do not have to worry about numpy types.  We control the conversion from proto into the gate:
https://github.com/quantumlib/Cirq/blob/master/cirq-google/cirq_google/serialization/arg_func_langs.py#L199
No need to worry about user input, just our own internal inconsistencies!